oms-kemp-applicationdelivery-solution/README.md

```diff
@@ -0,0 +1,38 @@
+# OMS Kemp Application Delivery Solution
+
+## Prerequisites
+
+The solution displays data collected from the Kemp devices through an extension for [OMS agent for Linux](https://github.com/Microsoft/OMS-Agent-for-Linux). The instructions on how to install and configure the agent extension are on a separate [github repo](https://github.com/QuaeNocentDocent/omskemp)
+
+This solution will display status, assets and performance data from your [Kemp](www.kemptechnologies.com) Application Delivery (was loadmaster) in your OMS Log Analytics workspace.
+
+![SolutionOverview](images/overview.png?raw=true)
+
+
+## Installation
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Foms-kemp-applicationdelivery-solution%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Foms-kemp-applicationdelivery-solution%2Fazuredeploy.json" target="_blank">
```

Comment:
know of the case sensitiveness, just cannot see where it is wrong. Tried substituting my own repo on github and it works. Any clue?